modules/ztp-preparing-the-hub-cluster-for-ztp.adoc

```diff
@@ -8,103 +8,50 @@
 
 You can configure your hub cluster with a set of ArgoCD applications that generate the required installation and policy custom resources (CR) for each site based on a zero touch provisioning (ZTP) GitOps flow.
 
+.Prerequisites
+
+* Openshift Cluster 4.8 or 4.9 as the hub cluster
+* {rh-rhacm-first} operator 2.3 or 2.4 installed on the hub cluster
+* Red Hat OpenShift GitOps operator 1.3 on the hub cluster
+
 .Procedure
 
-. Install the Red Hat OpenShift GitOps Operator on your hub cluster.
+. Install the Topology Aware Lifecycle Operator, which coordinates with any new sites added by ZTP and manages application of the PGT-generated policies.
```

Comment:
It may be beyond the scope of this document, but can we link to instructions on how to install TALO? I haven't installed it manually and I'm still unclear as to how it would be accomplished. I assume through the operator hub?

Reply:
We will be providing these links in the final draft.